deployments/README.md

```diff
@@ -119,12 +189,7 @@ GRAFANA_ADMIN_PASS=admin_pass
 GRAFANA_SERVER_ROOT_URL=%(protocol)s://%(domain)s:%(http_port)s/
 GRAFANA_SERVER_DOMAIN=server.domain.io
 MATRIX_ACCESS_TOKEN="access-token"
-WITH_GIT=1   # Optional
 WITH_PROXY=1 # Optional
```

Comment:
I think our current deployment was actually using `GIT` version to make sure we always run the latest code, but I'm not 100% sure. I guess we can let it go though.

Reply:
No, it's using the nightly Docker images